draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1028,11 +1028,16 @@ POST https://\<log server>/ct/v2/add-chain
 
 Inputs:
 
+: precertificate:
+  : The base64 encoded precertificate.  This field MUST be included if and only
+    if the certificate is being submitted as a precertificate.
```

Comment:
In RFC6962, precertificates are X.509 certificates.  However, in 6962-bis, a precertificate is a CMS object containing a TBSCertificate.  So "certificate is being submitted as a precertificate" doesn't make sense.

How about shortening the description to ": The base64 encoded precertificate, if applicable." ?